src/RhinoInside.Revit/Convert.ToRhino.cs

```diff
@@ -466,8 +466,10 @@ public static Brep ToRhino(this DB.Face face, bool untrimmed = false)
 
         if (untrimmed)
         {
+#if REVIT_2018
```

Comment:
Hey @kike-garbo 
I usually use directives like `#if !(REVIT2013 || REVIT2014 || REVIT2015 || REVIT2016 || REVIT2017)` for backward compatibility. This way all versions above 2018 would pass this directive and use the new API.
Just a thought